packages/ckeditor5-emoji/tests/manual/emoji.md

```diff
@@ -1,5 +1,3 @@
-## foo
+## Emoji plugin
 
-### bar
-
-1. baz
+This test allows for testing of both `EmojiMention` and `EmojiPicker` plugins.
```

Comment:
We need to explain what to do to test these.

* For `EmojiMention`, a user needs to type `:` and then two letters. Also, opening a picker directly from the suggestions dropdown is possible.
* For `EmojiPicker,` a user can open a view that allows them to pick an emoji. Also, it would be good to check for integration with the menu bar feature.